website/docs/layouts/flex/partials/code/how-to-use.md

```diff
@@ -0,0 +1,573 @@
+  <Doc::placeholder @width="220px" @height="40px" @background="#e5ffd2">Some content</Doc::placeholder>
+  <Doc::placeholder @width="250px" @height="40px" @background="#e4c5f3">Other content</Doc::placeholder>
+  <Doc::placeholder @width="200px" @height="40px" @background="#d2f4ff">More content</Doc::placeholder>
+  <Doc::placeholder @width="300px" @height="40px" @background="#fff8d2">Extra content</Doc::placeholder>
+  <Doc::placeholder @width="240px" @height="40px" @background="#f3d9c5">Even more content</Doc::placeholder>
+</Hds::Layout::Flex>
+```
+In this case:
+
+- the `@wrap=\{{true}}` applied to the flex container allows the flex items to wrap on multiple lines
+- the `@gap=\{{array "8" "24"}}` applied to the flex container defines the gap between the items along the two "directions" (the first value is the space between "rows", the second between "columns")
+
+### Nesting
+
+It's possible to nest flexbox containers to achieve more complex layouts:
+
+```handlebars
+<Hds::Layout::Flex @direction="column" class="doc-flex-outlined-container">
+  <Doc::placeholder @height="50px" @background="#e5ffd2">Top content</Doc::placeholder>
+  <Hds::Layout::Flex @direction="row" as |LF|>
+    <Doc::placeholder @width="150px" @height="auto" @background="#e4c5f3">Side content</Doc::placeholder>
+    <LF.Item @grow={{true}}>
+      <Doc::placeholder @width="auto" @height="250px" @background="#d2f4ff">Main content</Doc::placeholder>
+    </LF.Item>
+  </Hds::Layout::Flex>
+  <Doc::placeholder @height="50px" @background="#fff8d2">Bottom content</Doc::placeholder>
+</Hds::Layout::Flex>
+```
+
+In this case:
+
+- the outer container is a column-based flexbox layout, with three child items
+- the second item is a row-based flexbox layout, used to create two side-by side content blocks; of these two the first has a fixed width, while the second fills the remaining area thanks to the `@grow=\{{true}}` attribute
+
+!!! Insight
+
+Depending on the complexity of the design you need to implement, you may want to consider using a CSS `grid` layout, instead of a `flex` layout.
+
+!!!
+
+---
+
+## Common layout patterns
+
+Below some more realistic examples of common layouts that can be achieved using the `Layout::Flex` component, in composition with other HDS components.
+
+!!! Warning
+
+**Important**
+
+The examples below are meant to show how one _could_ use the `Layout::Flex` component to implement certain common/standard UI patterns. They're **not** meant to be taken literally as they are and be used in production code. Also, some of these patterns are already implemented directly in HDS components that are ready to use.
+
+!!!
+
+### Media + Text/Content
+
+This is a classic layout, where a "media" element (it could be an image, an icon, an avatar, etc) is laid out next to a block of text (but could be more complex content). The two elements are visually centered along their horizontal axis.
```

Comment:
Just a note: The [classic "Media object"](https://www.stubbornella.org/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) referred to a media element laid out next to associated text with both aligned to the top.

I've sometimes seen Media objects with a centered horizontal axis layout referred to as "Flag objects" (really just a variation of the Media Object).

Reply:
I'll update the description:
```suggestion
This is a classic layout, where a "media" element (it could be an image, an icon, an avatar, etc) is laid out next to a block of text (but could be more complex content). The two elements are either visually centered along their horizontal axis, or aligned to their top.
```